WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsJetpackSocialContainerView.kt

```diff
@@ -0,0 +1,121 @@
+package org.wordpress.android.ui.posts
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import org.wordpress.android.models.PublicizeConnection
+import org.wordpress.android.models.PublicizeConnectionList
+import org.wordpress.android.ui.compose.components.TrainOfIconsModel
+import org.wordpress.android.ui.compose.components.buttons.PrimaryButton
+import org.wordpress.android.ui.compose.theme.AppThemeEditor
+import org.wordpress.android.ui.compose.unit.Margin
+import org.wordpress.android.ui.posts.social.PostSocialConnection
+import org.wordpress.android.ui.posts.social.compose.PostSocialMessageItem
+import org.wordpress.android.ui.posts.social.compose.PostSocialSharesText
+import org.wordpress.android.ui.publicize.PublicizeServiceIcon
+
+@Composable
+fun EditPostSettingsJetpackSocialContainerView(
+    trainOfIconsModels: List<TrainOfIconsModel>,
+    postSocialConnectionList: List<PostSocialConnection>,
+    shareMessage: String,
+    remainingSharesMessage: String,
+    subscribeButtonLabel: String,
+    onSubscribeClick: () -> Unit,
+    connectProfilesButtonLabel: String,
+    onConnectProfilesCLick: () -> Unit,
```

Comment:
💡 Maybe it's worth receiving `modifier` as well, especially since callers of this Composable might want it to fill the entire width.

Reply:
Makes sense 👍 thanks